### DISCUSSION TITLE
Getting started as a contributor

### DISCUSSION THREAD

**Discussion by robpc** [spuithori/tokimekibluesky#101] [General]
What is the best way to get started as a potential contributor? I cloned the repo and tried running it locally [as described in the README.md](https://github.com/spuithori/tokimekibluesky?tab=readme-ov-file#how-to-build), but I am getting a 500 error like below. I am guessing that there is more to the the settings in `.env` than copying the example. 

<img width="722" alt="Screenshot 2024-11-16 211032" src="https://github.com/user-attachments/assets/22b51135-1bf3-48ae-80c3-ab3834f9f966">

**Comment by robpc** [comment]
Actually I just saw this commit and pulling latest works for me https://github.com/spuithori/tokimekibluesky/commit/9c3f7e427d51d2edc9472c848146c3ff09ef8f62

**Reply by spuithori** [reply]
Thank you very much!
Yes, I fixed the .env as you pointed out, but I forgot to make an announcement.
